lib/sigevent.c

```diff
@@ -250,6 +250,8 @@ core_handler(int signo, siginfo_t *siginfo, void *context)
 
 	/* dump memory stats on core */
 	log_memstats(stderr, "core_handler");
+
+	zlog_tls_buffer_fini();
```

Comment:
did the `buffer_flush()` api not work here? you don't really care about the resources involved, right - just flushing any buffered messages might be lighter, most likely to succeed if something has gone this wrong

Reply:
it did not work nearly as good.... Why?  Not sure.  But I was still missing data that is there when I use _fini()

Reply:
ok, thanks